src/browser/BrowserOptionDialog.cpp

```diff
@@ -47,6 +47,12 @@ BrowserOptionDialog::BrowserOptionDialog(QWidget* parent) :
     connect(m_ui->useCustomProxy, SIGNAL(toggled(bool)), m_ui->customProxyLocationBrowseButton, SLOT(setEnabled(bool)));
     connect(m_ui->customProxyLocationBrowseButton, SIGNAL(clicked()), this, SLOT(showProxyLocationFileDialog()));
 
+#ifdef Q_OS_WIN
+    // Vivaldi uses Chrome's registry settings
+    m_ui->vivaldiSupport->setChecked(false);
+    m_ui->vivaldiSupport->setEnabled(false);
```

Comment:
Perhaps we should hide it instead of just disabling it.

Reply:
That would be better, yes.